exercise/exercise_1.js

```diff
@@ -1,46 +1,135 @@
+
+function fibonacci(m){
+  if (m==1 || m==2) {return 1;}
+  else {return fibonacci(m-1)+fibonacci(m-2);} 
+}
+
 /////////////////////////////////////////////////////Exercise 1//////////////////////////////////////////////////
   // Write function have one paramiter(n). Return the sequence of Fibonacci numbers in the interval n.
   // input: 4
-  // output:1 1 2 3
-  
-  // Code
-  
+  // output:1 1 2 3 
+ 
+  function fibonacci_numbers(n){
+    var arr=[]; var i;
+    for (i=0;i<n;i++)
+      arr[i]=fibonacci(i+1);
+    for (i=0;i<n;i++)
+      console.log(arr[i]);}
+  fibonacci_numbers(4);
```

Comment:
function fibonacci_numbers(n){
    var arr=[]; var i;
    for (i=0;i<n;i++)
      arr[i]=fibonacci(i+1);
    return arr;
    }